### DISCUSSION TITLE
Did not find metrics : milvus_datanode_unflushed_segment_num

### DISCUSSION THREAD

**Discussion by navneetrajput06** [milvus-io/milvus#36239] [Q&A and General discussion]
Hi Team,
We are on milvus 2.4.4 but did not see metrics: milvus_datanode_unflushed_segment_num though this metrics mentioned in milvus documentation.
<img width="1734" alt="Screenshot 2024-09-12 at 10 44 22 PM" src="https://github.com/user-attachments/assets/96f31623-99eb-44a5-811f-597aa9c75486">

**Comment by yhmo** [comment]
Wait @LoveEachDay comment.

**Reply by LoveEachDay** [reply]
@navneetrajput06 The metrics of `milvus_datanode_unflushed_segment_num` has been removed since milvus 2.2. To learn the segment state, you can refer to  metric `milvus_datacoord_segment_num`. Please be note the label `segment_level` and `segment_state`.

`segment_state` has the following values:
- Growing
- Sealed
- Flushing
- Flushed
- Dropped

`segment_level` has the following values:
- L0
- L1
- Legacy
